sdk/template/template/samples/javascript/sample.env

```diff
@@ -1,10 +1,7 @@
-# Used in most samples. Retrieve these values from a Cognitive Services instance
```

Comment:
I do not think we need to have multiple `.env` files.

Reply:
These are for the samples. They need to be there in addition to the one in the package root, because these are shipped with the samples on docs.ms